nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/controller-service/controller-service-table/controller-service-table.component.html

```diff
@@ -116,62 +116,73 @@
-                                <div
-                                    class="pointer fa fa-cogs primary-color"
-                                    (click)="advancedClicked(item, $event)"
-                                    title="Advanced"></div>
+                    <div class="flex items-center justify-end gap-x-2">
+                        <button
+                            mat-icon-button
+                            type="button"
+                            [matMenuTriggerFor]="actionMenu"
+                            class="h-16 w-16 flex items-center justify-center icon global-menu">
+                            <i class="fa fa-ellipsis-v"></i>
+                        </button>
+                        <mat-menu #actionMenu="matMenu" xPosition="before">
+                            @if (definedByCurrentGroup(item)) {
+                                <button mat-menu-item (click)="configureClicked(item, $event)">
+                                    <i class="fa fa-cog primary-color mr-2"></i>
+                                    {{ canConfigure(item) ? 'Edit' : 'View Configuration' }}
+                                </button>
+                                @if (hasAdvancedUi(item)) {
+                                    <button mat-menu-item (click)="advancedClicked(item, $event)">
+                                        <i class="fa fa-cogs primary-color mr-2"></i>
+                                        Advanced
+                                    </button>
+                                }
+                                @if (canDisable(item)) {
+                                    <button mat-menu-item (click)="disableClicked(item)">
+                                        <i class="icon icon-enable-false primary-color mr-2"></i>
+                                        Disable
+                                    </button>
+                                }
+                                @if (canEnable(item)) {
+                                    <button mat-menu-item (click)="enabledClicked(item)">
+                                        <i class="fa fa-flash primary-color mr-2"></i>
+                                        Enable
+                                    </button>
+                                }
+                                @if (canChangeVersion(item)) {
+                                    <button mat-menu-item (click)="changeVersionClicked(item)">
+                                        <i class="fa fa-exchange primary-color mr-2"></i>
+                                        Change Version
+                                    </button>
+                                }
+                                @if (canDelete(item)) {
+                                    <button mat-menu-item (click)="deleteClicked(item, $event)">
+                                        <i class="fa fa-trash primary-color mr-2"></i>
+                                        Delete
```

Comment:
Clicking the delete or edit action does not close the menu:
<img width="863" alt="Screenshot 2024-05-07 at 5 09 18 PM" src="https://github.com/apache/nifi/assets/713866/7f2fcc7f-b493-4cab-8474-efa4582364de">

Reply:
This happens anywhere we `event.stopPropagation();`. I think these were in place to not cause selection to fire. I will remove them.